src/marionnette_extensions/animatable_region.coffee

```diff
@@ -23,8 +23,7 @@ class AnimatableRegion extends Marionette.Region
 
   goHome: (route, transition) ->
     @navigationStack.clear()
-    @navigationStack.unshift { route: route, transition: transition }
-    @goBack(route)
+    @goBack route, { transition: transition }
```

Comment:
To me it seems this is the only change you need to fix the problem you were describing earlier. Here indeed the transition argument was not passed to `@goBack` so this change make sense.

Can you try in your specific case if this change alone indeed fixes your problem? If not perhaps we could have a screen share session soon to look into the issue in your specific case.